paies/views.py

```diff
@@ -0,0 +1,172 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponse
+from django.utils import timezone
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import padding
+from django.views.decorators.csrf import csrf_exempt
+from django.conf import settings
+import hashlib
+import binascii
+
+
+MerchantID = settings.ENCRYPTION_KEY['MERCHANT_ID']
+HASHKEY = settings.ENCRYPTION_KEY['HASH_KEY']
+HASHIV = settings.ENCRYPTION_KEY['HASH_IV']
+Version = settings.ENCRYPTION_KEY['VERSION']
+ReturnUrl = settings.ENCRYPTION_KEY['RETURN_URL']
+NotifyUrl = settings.ENCRYPTION_KEY['NOTIFY_URL']
+PayGateWay = settings.ENCRYPTION_KEY['PAY_GATEWAY']
+RespondType = settings.ENCRYPTION_KEY['RESPOND_TYPE']
+
+orders = {}
+def index(request):
+    return render(request, 'paies/index.html', {'title': 'Express'})
+
+@csrf_exempt
+def create_order(request):
+    if request.method == 'POST':
+        data = request.POST
+
+        # 使用 Unix Timestamp 作為訂單編號（金流也需要加入時間戳記）
+        timestamp = timezone.now().timestamp()
+        order = {
+            **data,
+            'MerchantID': MerchantID,
+            'RespondType': 'JSON',
+            'TimeStamp': timestamp,
+            'Version': 2.0,
+            'Amt': int(100),
+            'MerchantOrderNo': timestamp,
```

Comment:
```suggestion
            'MerchantOrderNo': timestamp,
```
訂單編號如果多筆訂單在同一秒建立，會不會有相同的 id？

Reply:
想提問～ 這部分有找到解決辦法嗎～

Reply:
我也想了解?